R/utils.R

```diff
@@ -1,3 +1,14 @@
 `%||%` <- function(a, b) { # nolint
   if (is.null(a)) b else a
 }
+
+#' @noRd
+remove_non_numerics <- function(l) {
+  clean <- list()
+  for (key in names(l)) {
+    if (class(l[[key]]) %in% c("numeric", "integer", "array")) {
```

Comment:
Given `array` could be an array of characters, do you want to test instead

```
if (storage.mode(l[[key]]) %in% c("integer", double")) {
```

Reply:
thanks! Added a test